examples/scripts/compute_reference_Morris_indices.py

```
@@ -1,38 +1,95 @@
+    sample = ot.Sample(dimension, 3)
+    sample[:, 0] = ot.Sample.BuildFromPoint(point1)
+    sample[:, 1] = ot.Sample.BuildFromPoint(point2)
+    for i in range(dimension):
+        sample[i, 2] = abs(point1[i] - point2[i])
+    sample.setDescription(["Reference", "Computed", "Difference"])
+    return sample
+
+# %%
+print("First order indices")
+first_order_table = comparisonSample(reference_first_order, result.first_order_indices)
+first_order_table
```

Ruff (0.15.15)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)